Jenkinsfile

```diff
@@ -143,6 +145,7 @@ builders['msvc2015'] = {node('windows'){
               }
               junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/results.xml'
               step([$class: 'ArtifactArchiver', artifacts: "build_${version}/veles-${version}.zip", fingerprint: true])
+              bat(script: "rd /s /q build_${version}", returnStatus: true)
```

Comment:
rant. eh, someday we need to either merge both msvc builders to really use version variable, or remove it all together :P